resources/tests/msr/msr_reader.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Helper script used to read MSR values from ranges known to contain MSRs.
+
+print_msr() {
+    local msr_hex=$(printf "%#x" $1)
+    if output=$(rdmsr $msr_hex 2>> /dev/null); then
+        echo "$msr_hex,implemented,$output"
+    else
+        echo "$msr_hex,unimplemented,NaN"
+    fi
+}
+
+echo "MSR_ADDR,STATUS,VALUE"
+
+# 0x0..0xfff
+for((msr=0;msr<=4095;msr++))
+do
+    print_msr $msr
+done
+
+# 0x10000..0x10FFF
+for((msr=65536;msr<=69631;msr++))
```

Comment:
I looked up and you could use hex directly `((msr=16#10000;msr<=16#10FFF;msr++))`

Reply:
Changed to hex values.